site/en/docs/extensions/mv3/intro/mv3-migration/index.md

```diff
@@ -355,106 +339,109 @@ chrome.tabs.executeScript({
 ```
 
 ```js
-// Manifest v3
+// Manifest V3
 
 // background.js
+async function getCurrentTab() {/* ... */}
+let tab = await getCurrentTab();
+
 function showAlert() {
-  alert("test!");
+  alert('Function test alert');
 }
 
 chrome.scripting.executeScript({
+  target: {tabId: tab.id},
   function: showAlert
```

Comment:
Same comment about this now being called `func:`.

Reply:
Ah, I didn't realize this landed in stable. Thanks :)